h5bp/web_performance/cache-control.conf

```diff
@@ -0,0 +1,33 @@
+# ----------------------------------------------------------------------
+# | Cache Control                                                      |
+# ----------------------------------------------------------------------
+
+# TBD
+#
+# (1) TBD
+#
+# (2) TBD
+#
+# (3) TBD
+#
+# https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control
+
+map $sent_http_content_type $cache_control {
+  # (1)
+  default           "private, must-revalidate";
+
+  # (2)
+  ~*text/html       "public, immutable";
+  ~*text/css        "public, immutable";
+  ~*text/javascript "public, immutable";
+  ~*xml             "public, immutable";
```

Comment:
This would also match atom/rss feeds, which are very much not immutable.

XML seems like it could be any number of things (like data used in a JS app), and therefore probably not safe to assume it's public. I'd suggest matching atom/rss specifically to be public.

Reply:
Definitely a good point!